README.md

```diff
@@ -145,6 +153,165 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 4. Distributed Events
+
+This chapter introduces Distributed Events which are Cluster Events and Distibuted Data Structure Events.
```

Comment:
This chapter explains when various events are fired and describes how you can add event listeners on a Hazelcast Node.js client. These events can be categorized as cluster and distributed data structure events.